app/src/components/governance/ProposalStatusBar.vue

```diff
@@ -31,18 +31,14 @@
           <span>{{ votePercentage | percentInt }} of {{ stakingDenom }}</span>
           <span v-if="voteCount">({{ voteCount }} Votes)</span>
         </div>
-        <div v-if="true" class="vote-data">
+        <div v-if="proposal.type === governanceStatusEnum.TREASURY" class="vote-data">
```

Comment:
i don't understand why thresholds are only for treasury proposals...? every proposal type has a threshold.